packages/core/utils/lib/__tests__/async.test.js

```diff
@@ -77,4 +77,48 @@ describe('Async utils', () => {
       expect(maxOperations).toEqual(2);
     });
   });
+  describe('reduceAsync', () => {
+    test('Should return a incremented number', async () => {
+      const numberPromiseArray = [Promise.resolve(1), Promise.resolve(2)];
+
+      const reduceFunc = reduceAsync(numberPromiseArray);
+      const result = await reduceFunc(
+        (previousValue, currentValue) => previousValue + currentValue,
+        10
+      );
+
+      expect(result).toEqual(13);
+    });
+    test('Should work with mix of promises and values', async () => {
+      const numberMixArray = [1, Promise.resolve(2)];
+
+      const reduceFunc = reduceAsync(numberMixArray);
+      const result = await reduceFunc(
+        (previousValue, currentValue) => previousValue + currentValue,
+        10
+      );
+
+      expect(result).toEqual(13);
+    });
+    test('Should throw an error', async () => {
+      const numberPromiseArray = [Promise.resolve(1), Promise.resolve(2)];
+
+      const reduceFunc = reduceAsync(numberPromiseArray);
+
+      await expect(async () => {
+        await reduceFunc(() => {
+          throw new Error('test');
+        }, null);
+      }).rejects.toThrow('test');
+    });
+    test('Should throw an error 2', async () => {
```

Comment:
Do you see a test name that would be more explicit on what is tested? Same for above

Reply:
Should be good now 🙂

Reply:
Nice, I understand better thanks :)